src/data/faq_de.json

```
@@ -335,6 +335,22 @@
                             "Beachten Sie, dass die App  maximal einen Schnelltest und einen PCR-Test gleichzeitig verwalten kann. Wenn Sie beispielsweise einen PCR-Test wiederherstellen, und zuvor einen weiteren PCR-Test registriert hatten, wird der aktive PCR-Test in den Papierkorb verschoben.",
                             "Zur Veranschaulichung können Sie unter den folgenden Links Screenshots zum Papierkorb für sowohl iOS als auch Android Geräte finden:<ul><li><a href='../screenshots/#ios_recycle_bin' target='_blank'>iOS Screenshots: Papierkorb</a></li><li><a href='../screenshots/#android_recycle_bin' target='_blank'>Android Screenshots: Papierkorb</a></li></ul>"
                         ]
+                    },
+                    {
+                        "title": "Fragen und Antworten zum aktuellen Statusnachweis",
+                        "anchor": "admission_policy",
+                        "active": false,
+                        "textblock": [
+                            "Die Corona-Warn-App zeigt Ihnen in der Übersicht der Zertifikate für jede Person, für die Sie Zertifikate hinterlegt haben, einen Eintrag an. Neben dem Namen der Person wird in einem blauen Punkt die Regel angezeigt, welche die Zertifikate der jeweiligen Person erfüllen. Hierbei gibt es derzeit vier verschiedene Möglichkeiten, nämlich 3G, 3G+, 2G und 2G+. Die Corona-Warn-App zeigt Ihnen immer die Regel an, die durch die hinterlegten Zertifikate höchstens erfüllt wird.",

```

Make it clearer that this was implemented in version 2.16.

```suggestion
                            "Die Corona-Warn-App zeigt Ihnen in der Übersicht der Zertifikate für jede Person, für die Sie Zertifikate hinterlegt haben, einen Eintrag an. Neben dem Namen der Person wird, ab Version 2.16, in einem blauen Punkt die Regel angezeigt, welche die Zertifikate der jeweiligen Person erfüllen. Hierbei gibt es derzeit vier verschiedene Möglichkeiten, nämlich 3G, 3G+, 2G und 2G+. Die Corona-Warn-App zeigt Ihnen immer die Regel an, die durch die hinterlegten Zertifikate höchstens erfüllt wird.",
```